task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -277,6 +278,12 @@ class WatchedSubprocess:
     # does not hang around forever.
     failed_heartbeats: int = attrs.field(default=0, init=False)
 
+    # Maximum possible time (in seconds) that task will have for execution of auxiliary processes
+    # like listeners after task is complete.
+    # TODO: This should come from airflow.cfg: [core] task_success_overtime
+    task_overtime_threshold: ClassVar[float] = 20.0
```

Comment:
```suggestion
    TASK_OVERTIME_THRESHOLD: ClassVar[float] = 20.0
```

perhaps?

Reply:
I originally did that but saw `procs` was lower-case so changed it back. But I like upper-cased constants. will change it

Reply:
Done in https://github.com/apache/airflow/pull/44590/commits/5b8fd679f213f98a6828aec8f370176fdca7c8c1

Reply:
I'm removing procs in my next PR anyway, it's not needed.